test/functional/tools/column_param.xml

```diff
@@ -1,10 +1,13 @@
 <tool id="column_param" name="Column Param">
-  <command>
-    cut -f '$col' '$input1' > '$output1'
-  </command>
+  <command><![CDATA[
+    echo "col $col" >> '$output1' &&
```

Comment:
This is odd to add to that file right - can you echo these to a new second output?

Reply:
Good idea.